daemon/src/model/cfd.rs

```diff
@@ -390,10 +391,15 @@ impl Cfd {
         Ok(margin)
     }
 
-    pub fn profit(&self, current_price: Usd) -> Result<(Amount, Usd)> {
-        let profit =
-            calculate_profit(self.order.price, current_price, dec!(0.005), Usd(dec!(0.1)))?;
-        Ok(profit)
+    pub fn profit(&self, current_price: Usd) -> Result<(Decimal, Decimal)> {
```

Comment:
I think the profit calculation has to be different for taker and maker - at the moment we use the same formula, does that make sense?

Reply:
Why should it be different? As long as we fill in the correct values (maker's margin), it should be correct.

Reply:
But they have different positions, and the position is not taken into account in the calculation. Wouldn't this result in always displaying a positive profit or negative profit for both?

Reply:
Fixed in the last commit.